app/services/collators/disposable_income_collator.rb

```diff
@@ -19,14 +19,16 @@ class DisposableIncomeCollator
              :gross_employment_income, to: :@gross_income_summary
 
     class << self
-      def call(disposable_income_summary:, gross_income_summary:)
-        new(gross_income_summary:, disposable_income_summary:).call
+      def call(disposable_income_summary:, gross_income_summary:, eligible_for_partner_allowance:, submission_date: nil)
+        new(gross_income_summary:, disposable_income_summary:, eligible_for_partner_allowance:, submission_date:).call
```

Comment:
I think this is telling us that we should just be passing the partner allowance rather than 2 things that allow it to be calculated (one of which is optional)

Reply:
Good point. Will change.